devito/finite_differences/finite_difference.py

```diff
@@ -224,10 +232,10 @@ def second_cross_derivative(function, dims, order):
     :param order: Discretisation order of the stencil to create.
     """
     first = second_derivative(function, dim=dims[0], width=order)
-    return second_derivative(first, dim=dims[1], order=order).evalf(_PRECISION)
+    return Add(second_derivative(first, dim=dims[1], order=order).evalf(_PRECISION))
```

Comment:
feels like it's rather ``second_derivative`` that should already return an ``Add``